### ISSUE TITLE
Is there a way to run scrcpy in wireless mode with both phone and computer being on the same network?

### ISSUE THREAD

**Issue by ceyhun242** [Genymobile/scrcpy#2086]
Is it possible without writing any code to command the prompt but just clicking on scrcpy automatically running the application in wireless mode or prompts the user to choose between USB or wireless maybe?

Thank you

**Comment by rom1v** [comment]
No, wireless is described here: https://github.com/Genymobile/scrcpy#wireless

**Comment by ceyhun242** [comment]
According to the message below, it sounds possible no? what did you say no to? can I add a code to the target bar in the shortcut properties so that it will always connect with wireless at home?

Thanks

"Autostart on device connection
You could use AutoAdb:

autoadb scrcpy -s '{}'"

**Comment by heezes** [comment]
- `adb start-server
- adb tcpip 5555
- adb connect $DEVICE_IP:5555
- scrcpy`
follow the above mentioned sequence and u could connect to the device remotely.
NOTE: You have to connect the phone to pc via usb once and configure the wireless connection.

**Comment by heezes** [comment]
```
- adb start-server
- adb tcpip 5555
- abd connect $DEVICE_IP:5555
- scrcpy
```

**Comment by moxzie** [comment]
Watch my video here on how to connect it wirelessly..
http://bit.ly/2EEoJog

**Comment by ceyhun242** [comment]
> ```
> - adb start-server
> - adb tcpip 5555
> - abd connect $DEVICE_IP:5555
> - scrcpy
> ```

Thank you

**Comment by ceyhun242** [comment]
> Watch my video here on how to connect it wirelessly..
> http://bit.ly/2EEoJog

Thank you
